docs/source/sensors.rst

```diff
@@ -116,6 +116,8 @@ For example:
     self._logger = self._sensor_service.get_logger(name=self.__class__.__name__)
     self._logger.debug('Polling 3rd party system for information')
 
+.. _ref-sensors-datastore-management-operations
```

Comment:
add `:` at the end of ref so `.. .. _ref-sensors-datastore-management-operations:`

Would have patched it myself but too lazy to go through the process of forking your fork and then pushing out a PR :)